hugo/content/_index.md

```diff
@@ -1,11 +1,9 @@
 ---
 Title: ACL Anthology
-date: "2019-05-05"
+date: "0001-01-01"
```

Comment:
What is this date supposed to be? I don't know where it could be documented, maybe an inline comment?